### PR TITLE
Set up Saucelabs

### PR DESCRIPTION
**Pull request by m-mujica**

### CODE REVIEW

File: test/test-sauce-labs.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
+/* jshint esversion: 6 */
```
Comment (matthewp) [inline test/test-sauce-labs.js]: is this at the point where we can abstract this into its own module / repo?

Comment (m-mujica) [inline test/test-sauce-labs.js]: Yup, it's time I think https://blog.codinghorror.com/rule-of-three/ want me to tackle that as part of this PR?

Comment (matthewp) [inline test/test-sauce-labs.js]: yeah, what do you think @phillipskevin, bitovi/test-saucelabs ?

Comment (matthewp) [inline test/test-sauce-labs.js]: We don't have an org for testing stuff.

Comment (phillipskevin) [inline test/test-sauce-labs.js]: Yeah, I think that's a good idea. We'll have to extract out the platforms and maybe some other options into a json config probably.

**Comment by m-mujica** [comment]
@matthewp 👍  or 👎

**Comment by matthewp** [comment]
@m-mujica can you also create the test-saucelabs project as part of this overall issue? And then use it in steal-css?

**Pull request merged**